src/google/protobuf/util/internal/default_value_objectwriter.cc

```diff
@@ -411,16 +411,33 @@ void DefaultValueObjectWriter::MaybePopulateChildrenOfAny(Node* node) {
 DataPiece DefaultValueObjectWriter::FindEnumDefault(
     const google::protobuf::Field& field, const TypeInfo* typeinfo,
     bool use_ints_for_enums) {
-  if (!field.default_value().empty())
-    return DataPiece(field.default_value(), true);
-
   const google::protobuf::Enum* enum_type =
       typeinfo->GetEnumByTypeUrl(field.type_url());
   if (!enum_type) {
     GOOGLE_LOG(WARNING) << "Could not find enum with type '" << field.type_url()
                  << "'";
     return DataPiece::NullData();
   }
+  if (!field.default_value().empty()) {
+    if (!use_ints_for_enums) {
+      return DataPiece(field.default_value(), true);
+    } else {
+      std::string enum_default_value_name = field.default_value();
```

Comment:
You can make this a `const std::string&` to avoid making a copy here.

Reply:
done